flowmachine/tests/conftest.py

```diff
@@ -106,7 +106,7 @@ def skip_datecheck(request, monkeypatch):
 
 @pytest.fixture(autouse=True)
 def flowmachine_connect():
-    con = flowmachine.connect()
+    con = flowmachine.connect(db_pass="foo", redis_password="fm_redis")
```

Comment:
I'm fine with explicitly providing a password in the tests. Just thinking that an alternative could be to _require_ the env vars to be set instead (and abort the tests immediately if they are not). No strong opinion really which is better, but it would help us to avoid sprinkling dummy passwords in a number of places?